Meadowlark-js/backends/meadowlark-local-postgres-opensearch/src/BackendFacade.ts

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import {
+  DeleteRequest,
+  DeleteResult,
+  GetRequest,
+  GetResult,
+  QueryRequest,
+  QueryResult,
+  UpdateRequest,
+  UpdateResult,
+  UpsertRequest,
+  UpsertResult,
+} from '@edfi/meadowlark-core';
+
+// import * as Upsert from './repository/Upsert';
+// import * as Delete from './repository/Delete';
+// import * as Read from './repository/Read';
+import * as Query from './repository/Query';
+// import * as Update from './repository/Update';
+import { getSharedClient } from './repository/Db';
+
+// @ts-ignore
+export async function deleteDocumentById(request: DeleteRequest): Promise<DeleteResult> {
+  return { result: 'UNKNOWN_FAILURE', failureMessage: '' };
+  // return Delete.deleteDocumentById(request, await getSharedClient());
```

Comment:
This draft is for connection to Postgres and checking tables exist or creating them, all CRUD functions have been commented out and will be added soon